core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BpmBillingProjectLifecycle.scala

```diff
@@ -6,6 +6,7 @@ import bio.terra.profile.model.ProfileModel
 import bio.terra.workspace.model.CreateLandingZoneResult
 import cats.implicits.{catsSyntaxFlatMapOps, toTraverseOps}
 import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.rawls.billing.BillingProfileManagerDAO.ProfilePolicy
```

Comment:
you're going to want to hide whitespace while reviewing this file. there are actually very few changes here.